sdk/src/message_interface/message/wallet_message.rs

```diff
@@ -0,0 +1,247 @@
+    GetAccounts,
+    /// Consume an account method.
+    /// Returns [`Response`](crate::message_interface::Response)
+    CallAccountMethod {
+        /// The account identifier.
+        #[serde(rename = "accountId")]
+        account_id: AccountIdentifier,
+        /// The account method to call.
+        method: AccountMethod,
+    },
+    /// Backup storage. Password must be the current one, when Stronghold is used as SecretManager.
+    /// Expected response: [`Ok`](crate::message_interface::Response::Ok)
+    #[cfg(feature = "stronghold")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "stronghold")))]
+    Backup {
+        /// The backup destination.
+        destination: PathBuf,
+        /// Stronghold file password.
+        #[derivative(Debug(format_with = "OmittedDebug::omitted_fmt"))]
+        password: String,
+    },
+    /// Change the Stronghold password to another one and also re-encrypt the values in the loaded snapshot with it.
+    /// Expected response: [`Ok`](crate::message_interface::Response::Ok)
+    #[cfg(feature = "stronghold")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "stronghold")))]
+    ChangeStrongholdPassword {
+        #[derivative(Debug(format_with = "OmittedDebug::omitted_fmt"))]
+        #[serde(rename = "currentPassword")]
+        current_password: String,
+        #[derivative(Debug(format_with = "OmittedDebug::omitted_fmt"))]
+        #[serde(rename = "newPassword")]
+        new_password: String,
+    },
+    /// Clears the Stronghold password from memory.
+    /// Expected response: [`Ok`](crate::message_interface::Response::Ok)
+    #[cfg(feature = "stronghold")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "stronghold")))]
+    ClearStrongholdPassword,
+    /// Checks if the Stronghold password is available.
+    /// Expected response:
+    /// [`Bool`](crate::message_interface::Response::Bool)
+    #[cfg(feature = "stronghold")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "stronghold")))]
+    IsStrongholdPasswordAvailable,
+    /// Find accounts with unspent outputs
+    /// Expected response: [`Accounts`](crate::message_interface::Response::Accounts)
+    RecoverAccounts {
+        #[serde(rename = "accountStartIndex")]
+        /// The index of the first account to search for.
+        account_start_index: u32,
+        #[serde(rename = "accountGapLimit")]
+        /// The number of accounts to search for, after the last account with unspent outputs.
+        account_gap_limit: u32,
+        #[serde(rename = "addressGapLimit")]
+        /// The number of addresses to search for, after the last address with unspent outputs, in
+        /// each account.
+        address_gap_limit: u32,
+        #[serde(rename = "syncOptions")]
+        /// Optional parameter to specify the sync options. The `address_start_index` and `force_syncing`
+        /// fields will be overwritten to skip existing addresses.
+        sync_options: Option<SyncOptions>,
+    },
```

Comment:
order of docs and attributes ... is there a formatting rule for this that fixes this automatically?

Reply:
I wish there was :(